src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const oneDayRentCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  switch (true) {
```

Comment:
If the condition is not an iterative of many values, it is better to use `if`. (Lesson 'Switch True', 2:10)